### DISCUSSION TITLE
My extension suddenly started failing

### DISCUSSION THREAD

**Discussion by Power-Maverick** [microsoft/vscode-discussions#859] [Extension Development QnA] [answered]
My extension - [Dataverse DevTools](https://marketplace.visualstudio.com/items?itemName=danish-naglekar.dataverse-devtools) suddenly stopped working. When I try to run it locally it works. I have upgraded all the packages. I have also upgraded the tool to bundle the output using webpack. None of these seems to be working.

How can I tell what is wrong with the extension?

Seems like the activate method itself is not getting invoked cause I don't see anything on the telemetry as well.

**Comment by r3m0t** [comment, marked as answer]
Did you reproduce the issue? (By installing the vsix instead of running from the project)

Check all Output channels, eg Main, Window, Extension Host, as well as the Extensions panel as sometimes it shows warnings/errors on your extension.

You can use `--inspect-brk-extensions` flag to VS Code and it will wait for a debugger to attach. Then you can tell your debugger to stop on all exceptions.
